<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>790</width>
    <height>550</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Preferred">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>0</width>
    <height>0</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>790</width>
    <height>550</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="background">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>-20</y>
     <width>801</width>
     <height>571</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
   <property name="pixmap">
    <pixmap>Gui.jpg</pixmap>
   </property>
  </widget>
  <widget class="QLineEdit" name="wakeTime">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="sleepTime">
   <property name="geometry">
    <rect>
     <x>230</x>
     <y>80</y>
     <width>131</width>
     <height>31</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="eventName">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>180</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="timeNo">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>200</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>No</string>
   </property>
  </widget>
  <widget class="QPushButton" name="timeYes">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>200</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Yes</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="startTime">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>276</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="endTime">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>310</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="estimatedTime">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>370</y>
     <width>321</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="anotherYes">
   <property name="geometry">
    <rect>
     <x>140</x>
     <y>480</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Yes</string>
   </property>
  </widget>
  <widget class="QPushButton" name="anotherNo">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>480</y>
     <width>31</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>No</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="output">
   <property name="geometry">
    <rect>
     <x>410</x>
     <y>40</y>
     <width>351</width>
     <height>471</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="submitButton">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>410</y>
     <width>321</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Submit Event</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
